test/storage/cookie_storage_test.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class CookieStorageTest < Minitest::Test
+  def setup
+    @storage = storage = Verdict::Storage::CookieStorage.new
```

Comment:
Is it worth adding a cookie storage-like class to test against, or actionpack to the gem's gemfile to test the real thing? Defaulting to `cookies: {}` seems like behaviour that is only useful for testing.

Reply:
Good point, it looks like there's already some basic Rails tests for the railtie so I'll see if I can leverage the dummy app to add an "integration" test 👍

Reply:
I'd removed the defaults for `cookies` and added a crude `ensure_cookies_is_set` method to `set`/`get`/`remove` methods. Admittedly my "mock" of `CookieJar` is _basic_ but it does allow us to test with a `CookieJar` instance 🎉 

Let me know what you think.